samples/bluetooth/ipsp/src/main.c

```diff
@@ -38,7 +38,9 @@ struct in6_addr in6addr_mcast = MCAST_IP6ADDR;
 
 static struct in6_addr in6addr_my = MY_IP6ADDR;
 
-#define MY_PORT 4242
+#ifndef CONFIG_NET_CONFIG_MY_PORT
+#define CONFIG_NET_CONFIG_MY_PORT 4242
+#endif
```

Comment:
I don't think it is a good idea (or even allowed) to define CONFIG_ macros.
probably better to do this:

```
#ifdef CONFIG_NET_CONFIG_MY_PORT
#define MY_PORT CONFIG_NET_CONFIG_MY_PORT
#else
#define MY_PORT 4242
#endif
```

Also maybe remove CONFIG from the middle there?

Reply:
Right. Because CONFIG has a special meaning in relation to Kconfig, and it might trip up scripts and the like.
There is already a script that complains about the first CONFIG.